src/android/com/adobe/phonegap/push/PushConstants.java

```diff
@@ -7,6 +7,10 @@
   public static final String FOREGROUND = "foreground";
   public static final String TITLE = "title";
   public static final String NOT_ID = "notId";
+  public static final String GROUP_ID = "groupId";
+  public static final String TO_DELETE_ID = "toDeleteId";
```

Comment:
```suggestion
  public static final String NOT_ID_TO_DELETE = "notIdToDelete";
```

So that it is clear that we talk about a notification ID and not a group ID here.